AISKU/scripts/listCdnVersions.ps1

```diff
@@ -1,5 +1,6 @@
 param (
     [string] $container = $null,                        # Identify the container that you want to check blank == all
+    [string] $storeContainer = "cdn",                   # Identifies the destination storage account container
```

Comment:
Set default container to "cdn" without the -cdn switch.

For users that don't want to use a container they can pass ""